### PR TITLE
EO-10 Verificar que una posicion no puede ser creada si los campos requeridos no son llenados

### PR DESCRIPTION
**Pull request by lines17**
Titulo: EO-10 Verificar que una posicion no puede ser creada si los campos requeridos no son llenados

### CODE REVIEW

**Comment by lines17** [comment]
TestCase:
eo-01writtentc

TestCase Passed:
eo-01-testexecuted

File: src/main/java/org/umssdiplo/automationv01/core/managepage/Position/CreatePosition.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    @FindBy(id = "add-position")
+    private WebElement addPositionButton;
+
+    @FindBy(id = "save-position")
+    private WebElement savePositionButton;
+
+    @FindBy(id = "register-position-title")
+    private WebElement registerPositionTitle;
+
+    public void clickOnOrganizationalMenuOption(){
```
Comment (marcelogaray) [inline src/main/java/org/umssdiplo/automationv01/core/managepage/Position/CreatePosition.java:24]: change to clickOrganizationalMenuOption

File: src/test/java/org/umssdiplo/automationv01/stepdefinitionproject/StepsDefinitionCreatePosition.java [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    }
+
+    @And("^User is authenticated$")
+    public void userIsAuthenticated() throws Throwable {
+        login.setCredentials();
+    }
+
+    @And("^I click on \"([^\"]*)\" menu button$")
+    public void iClickOnMenuButton(String arg0) throws Throwable {
+
```
Comment (marcelogaray) [inline src/test/java/org/umssdiplo/automationv01/stepdefinitionproject/StepsDefinitionCreatePosition.java:30]: remove blank line

File: src/test/java/org/umssdiplo/automationv01/stepdefinitionproject/StepsDefinitionCreatePosition.java [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+        createPosition.clickOnOrganizationalMenuOption();
+    }
+
+    @And("^I click on \"([^\"]*)\" option$")
+    public void iClickOnOption(String arg0) throws Throwable {
+
```
Comment (marcelogaray) [inline src/test/java/org/umssdiplo/automationv01/stepdefinitionproject/StepsDefinitionCreatePosition.java:36]: remove blank line

File: src/test/java/org/umssdiplo/automationv01/stepdefinitionproject/StepsDefinitionCreatePosition.java [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+        createPosition.clickOnListPositionOption();
+    }
+
+    @And("^I click on \"([^\"]*)\" Button$")
+    public void iClickOnButton(String arg0) throws Throwable {
```
Comment (marcelogaray) [inline src/test/java/org/umssdiplo/automationv01/stepdefinitionproject/StepsDefinitionCreatePosition.java:41]: unused parameter, remove it.

File: src/test/resources/features/CreatePosition.feature [hunk 5]
```diff
@@ -0,0 +1,10 @@
+Feature: CreatePosition
+
+  Scenario: EO-10 Verify that a position cannot be created without required fields
+    Given 'SSI' page is loaded
+    And User is authenticated
```
Comment (marcelogaray) [inline src/test/resources/features/CreatePosition.feature:5]: use one step definitons to set credentials

File: src/test/resources/features/CreatePosition.feature [hunk 6]
```diff
@@ -0,0 +1,10 @@
...
+    And I click on "Estructura Organizacional" menu button
```
Comment (marcelogaray) [inline src/test/resources/features/CreatePosition.feature:6]: dont use "I"

Comment (marcelogaray) [reply src/test/resources/features/CreatePosition.feature:6]: Example

According to the test case it should be:
Given I loging to 'SMARTHOUSE' page with properly credentials
When go to 'Estructura Organizacional' on 'Header' page
And navigate to 'Gerencia General' option on 'Dashboar/Left Panel/etc...' page
And click over 'Eliminar' button on 'Genencia General' page
Then 'Area Gerencia General' title should be displayed

**Review by marcelogaray — CHANGES_REQUESTED** [review]
review observation

File: src/test/java/org/umssdiplo/automationv01/stepdefinitionproject/StepsDefinitionCreatePosition.java [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+import org.umssdiplo.automationv01.core.utils.LoadPage;
+
+public class StepsDefinitionCreatePosition {
+    private Login login;
+    private CreatePosition createPosition;
+
+    @Given("^'SSI' page is loaded$")
+    public void ssiPageIsLoaded() throws Throwable {
+        login = LoadPage.loginPage();
+        createPosition = new CreatePosition();
```
Comment (marcelogaray) [inline src/test/java/org/umssdiplo/automationv01/stepdefinitionproject/StepsDefinitionCreatePosition.java:20]: it dont need to be here.

**Review by marcelogaray — CHANGES_REQUESTED** [review]
review observations

**Pull request closed without merging**